activesupport/lib/active_support/testing/parallelization/worker.rb

```diff
@@ -78,6 +78,7 @@ def safe_record(reporter, result)
         end
 
         def after_fork
+          ActiveSupport::TestCase.parallel_worker_id = @number if defined?(ActiveSupport::TestCase)
```

Comment:
Hrm, when is `ActiveSupport::TestCase` not defined?

Also style nit, add a new line after this.

```suggestion
          ActiveSupport::TestCase.parallel_worker_id = @number if defined?(ActiveSupport::TestCase)
          
```